packages/amazonq/src/lsp/client.ts

```diff
@@ -80,11 +81,13 @@ export async function startLanguageServer(
         executable = [resourcePaths.node]
     }
 
+    const memoryWarnThreshold = 200 * processUtils.oneMB
     const serverOptions = createServerOptions({
         encryptionKey,
         executable: executable,
         serverModule,
         execArgv: argv,
+        warnThresholds: { memory: memoryWarnThreshold },
```

Comment:
Is the value here only 200MB because the workspace context part of the language-servers repo isn't doing the indexing? I guess once we fully enable inline through Flare we'd probably have to re-adjust these numbers?

Reply:
Followed up with local indexing team, and it looks like it was disabled in my testing since this requires pulling the indexing server. After enabling its using ~700-800MB so I'll bump this to 1GB to be safe.